addons/mail/views/mail_mail_views.xml

```diff
@@ -26,7 +26,7 @@
                         </div>
                         <field name="mail_message_id_int" required="0" invisible="1"/>
                         <label for="subject" class="oe_edit_only"/>
-                        <h2><field name="subject"/></h2>
+                        <h2><field name="subject" class="w-75"/></h2>
```

Comment:
Have you tried using `colspan="2"` (or 3, or more) for this? It's generally better to use that I think.
Although this works if colspan doesn't work well.

Reply:
hi @thiryr ,
tried with colspan="2" and 3 for the field
and it seems having no effect,
is it fine if i move this field and label inside a div with class="oe_title", this seems to be working

Reply:
Yes, making it the title makes sense to me :+1:

Reply:
ok, I have modified the commit now